core/templates/pages/exploration-player-page/learner-experience/conversation-skin.directive.ts

```diff
@@ -402,13 +404,14 @@ angular.module('oppia').directive('conversationSkin', [
             DEFAULT_TWITTER_SHARE_MESSAGE_EDITOR,
             ENABLE_NEW_STRUCTURE_VIEWER_UPDATES,
             ENABLE_SOLICIT_ANSWER_DETAILS_FEATURE,
-            EVENT_NEW_CARD_AVAILABLE, EVENT_NEW_CARD_OPENED,
+            EVENT_NEW_CARD_OPENED,
             EXPLORATION_SUMMARY_DATA_URL_TEMPLATE, FEEDBACK_POPOVER_PATH,
             INTERACTION_SPECS, REVIEW_TESTS_URL_TEMPLATE,
             STORY_VIEWER_URL_TEMPLATE,
             SUPPORTED_SITE_LANGUAGES, TWO_CARD_THRESHOLD_PX,
             WHITELISTED_COLLECTION_IDS_FOR_SAVING_GUEST_PROGRESS) {
           var ctrl = this;
+          ctrl.directiveSubscriptions = new Subscription();
```

Comment:
Shouldn't this be unsubscribed in this file?

Reply:
Done